doc/guides/device_mgmt/index.rst

```diff
@@ -39,14 +39,417 @@ functionality over BLE and serial.
+      * ``baud``: the communication speed; must match the baudrate of the server.
+      * ``mtu``: aka Maximum Transmission Unit, the maximum protocol packet size.
+
+   .. group-tab:: ble
+
+      ``--connstring`` accepts the following ``key`` values:
+
+      * ``ctlr_name``: An OS specific string for the controller name.
+      * ``own_addr_type``: Can be one of ``public``, ``random``, ``rpa_pub``,
+        ``rpa_rnd``, where ``random`` is the default.
+      * ``peer_name``: The name the peer BLE device advertises, this should match
+        the configuration specified with :option:`CONFIG_BT_DEVICE_NAME`.
+      * ``peer_id``: The peer BLE device address or UUID. Only required when
+        ``peer_name`` was not given. The format depends on the OS where ``mcumgr``
+        is run, it is a 6 bytes hexadecimal string separated by colons on Linux, or
+        a 128-bit UUID on macOS.
+      * ``conn_timeout``: A float number representing the connection timeout in
+        seconds.
+
+   .. group-tab:: udp
+
+      ``--connstring`` takes the form ``[addr]:port`` where:
+
+      * ``addr`` can be a DNS name (if it can be resolved to the device IP),
+        IPv4 addr (app must be built with :option:`CONFIG_MCUMGR_SMP_UDP_IPV4`),
+        or IPv6 addr (app must be built with :option:`CONFIG_MCUMGR_SMP_UDP_IPV6`)
+      * ``port`` any valid IP port.
+
+Saving the connection config
+****************************
+
+The transport configuration can be managed with the ``conn`` sub-command and
+later used with ``--conn`` (or ``-c``) parameter to skip typing both ``--conntype``
+and ``--connstring``. For example a new config for a serial device that would
+require typing ``mcumgr --conntype serial --connstring dev=/dev/ttyACM0,baud=115200,mtu=512``
+can be saved with::
+
+  mcumgr conn add acm0 type="serial" connstring="/dev/ttyACM0,baud=115200,mtu=512"
+
+Accessing this port can now be done with::
+
+  mcumgr -c acm0
+
+List of Commands
+****************
+
+Not all commands defined by ``mcumgr`` (and SMP protocol) are currently supported
+on Zephyr. The ones that are supported are described in the following table:
+
+.. tip:: Running ``mcumgr`` with no parameters, or ``-h`` will display the list
+   of commands.
+
+.. list-table::
+   :widths: 10 30
+   :header-rows: 1
+
+   * - Command
+     - Description
+   * - ``echo``
+     - Send data to a device and display the echoed back data. This command is
+       part of the ``OS`` group which must be enabled by setting
+       :option:`CONFIG_MCUMGR_CMD_OS_MGMT`. The ``echo`` command itself can be
+       enabled by setting :option:`CONFIG_OS_MGMT_ECHO`.
+   * - ``fs``
+     - Access files on a device. More info in :ref:`fs_mgmt`.
+   * - ``image``
+     - Manage images on a device. More info in :ref:`image_mgmt`.
+   * - ``log``
+     - Currently unsupported. Used to manage logs on a device. Configuring
+       :option:`CONFIG_MCUMGR_CMD_LOG_MGMT` does not have any effect.
```

Comment:
The option no longer exists with merge of the c8ccf4fbc8bf8e9b3c3c61591dfae1a426e08969

Reply:
I removed it.